opensuse-tomcat-jul/pom.xml

```
@@ -0,0 +1,146 @@
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.cafapi.opensuse.tomcat</groupId>
+        <artifactId>opensuse-tomcat-image-aggregator</artifactId>
+        <version>5.7.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>opensuse-tomcat-jul</artifactId>
+    <packaging>pom</packaging>
+    
+    <name>openSUSE Tomcat jul</name>
+    <properties>
+        <maven.install.skip>true</maven.install.skip>
+        <DOCKER_HUB_PUBLIC>${dockerHubPublic}</DOCKER_HUB_PUBLIC>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.34.0</version>
+                <executions>
+                    <execution>
+                        <id>docker-build</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>start-docker-image</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>start</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>stop-docker-image</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>stop</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>docker-push</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>push</goal>
+                        </goals>
+                        <configuration>
+                            <filter>opensuse-tomcat-jul</filter>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <images>
+                        <image>
+                            <alias>opensuse-tomcat-jul</alias>
+                            <name>${dockerCafImagePrefix}opensuse-tomcat-jul${dockerProjectVersion</name>
```

```suggestion
                            <name>${dockerCafImagePrefix}opensuse-tomcat-jul${dockerProjectVersion}</name>
```
Missing closing } bracket.